selfdrive/ui/qt/util.h

```diff
@@ -18,6 +18,8 @@ QString getBrand();
 QString getUserAgent();
 std::optional<QString> getDongleId();
 QMap<QString, QString> getSupportedLanguages();
+
+[[deprecated]]
 void clearLayout(QLayout* layout);
```

Comment:
let's move it to cabana instead, want to make sure nothing else in the openpilot ui uses it